extensions/logging-json/runtime/src/main/java/io/quarkus/logging/json/runtime/JsonConfig.java

```diff
@@ -17,7 +17,7 @@
      * Determine whether to enable the JSON console formatting extension, which disables "normal" console formatting.
      */
     @ConfigItem(name = ConfigItem.PARENT)
-    boolean enabled;
+    boolean enable;
```

Comment:
This is not part of the test so it should be either a separate commit on top of the original commit that introduced the class or else it should be squashed into that commit.